Makefile

```diff
@@ -151,6 +151,14 @@ endif
 plugin-checker:
 	$(GO) run $(GOFLAGS) ./plugin/checker
 
+prepackaged-plugins:
```

Comment:
Can we add help text for this ala ` ## Populate the prepackaged-plugins directory`. Also, does this need a `.PHONY` directive?

Reply:
Ah yes Phony